code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -0,0 +1,404 @@
+/obj/structure/machinery/computer/shuttle/dropship/flight
+	name = "dropship navigation computer"
+	desc = "flight computer for dropship"
+	icon = 'icons/obj/structures/machinery/shuttle-parts.dmi'
+	icon_state = "console"
+	req_access = list(ACCESS_MARINE_LEADER, ACCESS_MARINE_DROPSHIP, ACCESS_WY_CORPORATE_DS)
+	unacidable = TRUE
```

Comment:
```suggestion
	unacidable = TRUE
	exproof = TRUE
```
Somebody put OB into FOB and destroyed the ground console. Not sure whether all of the type should be `exproof` (or even completely `indestructible`), or only some, or the entire `computer/shuttle` instead, but _some_ protection _somewhere_ is definitely in order.